lib/conf.c

```diff
@@ -224,8 +224,10 @@ parse_option(char *buf, const char **keyp, char **valp)
 	while (isblank((unsigned char)*value))
 		value++;
 
-	/* eat final newline */
-	value[strlen(value)-1] = '\0';
+	if (isspace((unsigned char)value[strlen(value)-1])) {
+		/* eat final newline */
+		value[strlen(value)-1] = '\0';
+	}
```

Comment:
I wonder if we could instead eat all whitespace from the end of the line?

And I think in this case it would be cleaner to store the string length in a variable instead of calling `strlen` twice. Why did you add the `unsigned char` cast as well?

Reply:
Disregard my question about the cast, though the need for it is ugh :p